### PR TITLE
changes for 1903

### PR DESCRIPTION
**Pull request by mbrown555**
removed 1709, added 1903

### CODE REVIEW

**Comment by mbrown555** [comment]
OK, I've made the requested changes

File: README.md [hunk 1]
```diff
@@ -13,21 +19,12 @@
...
-
 # Supported Windows Server 2016 amd64 tags
 
  `docker pull mcr.microsoft.com/windows/servercore/iis:windowsservercore-ltsc2016`
 
  [`windowsservercore-ltsc2016, latest` (*windowsserver/Dockerfile*)](https://github.com/Microsoft/iis-docker/blob/master/windowsservercore-ltsc2016/Dockerfile)
 
- `docker pull mcr.microsoft.com/windows/servercore/iis:nanoserver-sac2016`
-
-## What is IIS?
```
Comment (shirhatti) [inline README.md]: Looks like this line was erroneously deleted

Comment (mbrown555) [reply README.md]: crud, yep, I'll fix it

Comment (mbrown555) [reply README.md]: Fixed

File: windowsservercore-insider/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # escape=`
-FROM microsoft/windowsservercore-insider:latest
+FROM mcr.microsoft.com/windows/servercore:latest
```
Comment (shirhatti) [inline windowsservercore-insider/Dockerfile]:
```suggestion
FROM mcr.microsoft.com/windows/insider:10.0.18362.113
```

Comment (shirhatti) [reply windowsservercore-insider/Dockerfile]: Changed the insider image to point to the valid 19H1 image

**Pull request merged**